### ISSUE TITLE
Tweets can still be sent after the apps permissions are revoked from the twitter account

### ISSUE THREAD

**Issue by Softvision-GeluHaiduc** [mozilla/hubs#3640]
**[Affected Versions]:**
- Firefox Release 84.0.1
- Chrome 87.0.4280.88

**[Affected Platforms]:**
- Windows 10x64

**[Prerequisites]:**
- Have an account with Twitter permissions granted and logged in.   

**[Steps to reproduce]:**
1. Go to https://twitter.com/settings/connected_apps and “Revoke access” for hubs by Mozilla. 
2. Go to a room on https://hubs.mozilla.com and take a photo.
3. Click the “tweet” button from the generated picture inside the room.

**[Expected results]:**
- Permissions for the app need to be granted again.

**[Actual results]:**
- The tweet can be made, confirmation pop-up is displayed, but it is not posted on Twitter.

**[Notes]:**
- Hubs gives the impression that tweets can still be sent from the room even if the access is revoked.
- The issue persists in both old rooms and newly created rooms.
- There are no other options to grant permissions to hubs again.
- Attached a screen recording bellow
![revoke twitter](https://user-images.githubusercontent.com/51400741/103649010-6b46e980-4f66-11eb-8b14-9f4efdd26106.gif)

**Comment by Softvision-GeluHaiduc** [comment]
Possibly a duplicate of #1838

**Comment by brianpeiris** [comment]
Yup, this is a duplicate of #1838. We can close this or that one, or just close both of them when the issue is fixed.
